tests/update_workflow_test.go

```diff
@@ -5035,6 +5035,221 @@ func (s *UpdateWorkflowSuite) TestUpdateWorkflow_ContinueAsNew_UpdateIsNotCarrie
   4 WorkflowTaskCompleted`, s.GetHistory(s.Namespace(), tv.WorkflowExecution()))
 }
 
+func (s *UpdateWorkflowSuite) TestUpdateWithStart() {
```

Comment:
I moved these over here because (1) it's close to Update, so why not and (2) it can use the polling helpers here